src/managed/Microsoft.Extensions.DependencyModel/Microsoft.Extensions.DependencyModel.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <Description>Abstractions for reading `.deps` files.</Description>
     <TargetFrameworks>net451;netstandard1.3;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard1.3;netstandard1.6</TargetFrameworks>
```

Comment:
It looked like the issues around VS2017 hanging when you had conditions on the TargetFrameworks property have been fixed, but this pattern was recommended as a workaround before the fix goes everywhere.  When running in older versions on VS, it will use the unconditional version.

I didn't spend time trying to take package references to our private targeting packs, this felt more in line with what we expect customers to do at this point and I'd rather just adopt that pattern, unless you feel strongly.